### ISSUE TITLE
(feature request) Tracer: structured view of interaction params

### ISSUE THREAD

**Issue by kubukoz** [neandertech/langoustine#35]
It would be cool to show the params in a decoded form, with URIs potentially being clickable. I'm thinking a "nested table" view would be a good start.

**Comment by keynmol** [comment]
Yeah, we can go further than that even - I have a project somewhere that auto-generates Laminar UIs from case classes, I can merge it here with the data we have from langoustine itself, and give you both the clickable view, and the ability to convert it to a form and tweak the params a bit and see what the server will respond with.

**Comment by kubukoz** [comment]
That'd be really slick.

**Comment by keynmol** [comment]
Oh it's only been 4 months, but @kubukoz I'm working on the structured view (just figuring it out now) in #118.

Folding is to come, but at least I got some shitty rendering working (circe's API design is legit awesome) 
<img width="1509" alt="image" src="https://user-images.githubusercontent.com/1052965/208198008-d90bd83c-40be-4c4d-bc85-31b952a6a5cb.png">
